packages/report/package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "accessibility-insights-report",
-    "version": "4.2.2",
+    "version": "4.2.3",
```

Double checking: should this be a minor version instead of a patch?

```suggestion
    "version": "4.3.2",
```

---

Per chat, it sounds like we have a pattern of using patch.

---

Upon further discussion, we decided to go with minor version upgrade to reflect the changes in axe-core 4.4 that were more than bug fixes (there were some updates to at least one rule).